### DISCUSSION TITLE
reality怎么开chacha20

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [XTLS/Xray-core#2349] [Q&A]
没有cipherSuites字段了

**Comment by chika0801** [comment]
不清楚，当时合并了这PR没人研究怎么用，你不问的话都没人管这事

**Comment by RPRX** [comment]
1. **目标服务端选了哪个 Cipher Suite，REALITY 就会用哪个，这个是从一开始就有的、全自动的**
2. https://github.com/XTLS/REALITY/pull/4 **只是特定条件下用 ChaCha20 加密认证信息**（以前只用 AES），点进去看描述
**这个也是全自动的**，目前仅 `randomized` 指纹有概率触发，需要双端均为 Xray-core v1.8.3+
